mint/run/core/s3select/sql_ops.py

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# MinIO Python Library for Amazon S3 Compatible Cloud Storage,
+# (C) 2015-2020 MinIO, Inc.
```

Comment:
no need to add 2015, its not added in 2015 so should just say 2020

Reply:
> no need to add 2015, its not added in 2015 so should just say 2020

Fixed only here since this is a new file